Basic Newdle site (#249)
indico/newdle: index.md

> # Newdle

Reviewer: It's `newdle`, not `Newdle` (also below)

> **Newdle** is a tool developed by the [Indico Team](https://getindico.io/about/) @ [CERN](https://home.cern/) to **manage the process of organizing meetings**. As other commercial and Open Source solutions, Newdle provides the functionality of creating ad-hoc polls to schedule meetings when everybody (or at least most of the people involved) are able to attend. How is it possible? The tool provides for the creators of the polls (henceforth **newdles**) the possibility to choose different options to schedule the meeting, i.e. to choose different time slots within a day or from several days, in order to find the best options for everybody.

Reviewer 2: `s/How is it possibe/How is it done/` ?

> <img src="images/logo_color.svg" alt="hi" width="300" align="center" />

Reviewer: `alt="hi"`?! Let's put something more useful like "newdle logo" there... ;)

Once the newdle is done, the creator **sends the invitation to vote** to all the participants they want. Those participants will receive an email with the link of the newdle to vote and they can **choose those option(s) that fit** better for them. Once everyone has voted or the creator has closed the newdle it is **easy to choose the final date for the event**.

A few of the **most remarkable features** of this tool among others are the **user-friendly and modern interface** and the additional **freedom and flexibility**, features that come with being part of an Open Source ecosystem. Additionally, none of the other options available in the market seem to seamlessly integrate with other enterprise systems.

Newdle is part of the [MALT project](https://malt.web.cern.ch/malt/).

![A sneak peek of Newdle](images/sneakpeek.gif)

## Main features

- Add participants searching by name and/or email address
- Choose time slots for different days
- Copy time slots from previous day
- Keep list of participants private
- Check if I am part of any newdles

> - Check my newdles

Reviewer: This feels a bit low-level... What about something like this instead?

```
- Invite participants from an external user directory (e.g. LDAP)
- See the calendar availability of invited participants while defining dates/times to vote for
- Customizable privacy and notification settings
- Overview of the newdles you are involved with
- "If need be" choice for more fine-grained answers
- Automated cleanup of past newdles
```

<img src="images/logo_color.svg" alt="hi" width="300" align="center" />

Newdle has been created to streamline the process of choosing **the perfect date and time** for a meeting or event, avoiding the tedious and time consuming tasks that supposes organizing these meetings when is doing sending emails or private messages.

Once the newdle is done, the creator **sends the invitation to vote** to all the participants they want. Those participants will receive an email with the link of the newdle to vote and they can **choose those option(s) that fit** better for them. Once everyone has voted or the creator has closed the newdle it is **easy to choose the final date for the event**.

> One of the **most remarkable features** of this tool among others is the **user-friendly and modern interface** and also the additional **freedom and flexibility** that come with being part of an Open Source ecosystem. Additionally, none of the other options available in the market seem to seamlessly integrate with other enterprise systems.

Reviewer 3: ```suggestion
One of the **most remarkable features** of this tool among others is the **user-friendly and modern interface** and also the additional **freedom and flexibility** that comes with being part of an Open Source ecosystem. Additionally, none of the other options available in the market seem to seamlessly integrate with other enterprise systems.
```

I am not sure about this one, but somehoe `comes` feels like it is correct. What do the others think?

Reviewer 4: Agree. I think the problem with this sentence is that it starts with "One of..." and starts enumerating three features. Perhaps:
```suggestion
Few of the **most remarkable features** of this tool among others are the **user-friendly and modern interface** and the additional **freedom and flexibility** that comes with being part of an Open Source ecosystem. Additionally, none of the other options available in the market seem to seamlessly integrate with other enterprise systems.
```

Reviewer 2: "A few ..."

Reviewer 3: But with your changes it would be "come" again, right? 😖

Reviewer 4: As far as I know, I would say `comes` depends on whether it refers to the "flexibility" right (it) before or the remarkable features (they).

Reviewer 4: As far as I know, I would say "comes" depends on whether it refers to the "flexibility" right (it comes) before or the remarkable features (they come).

Reviewer: Let's have our native speaker haev a look ;) cc @hannahshort

Author: _freedom and flexibility, **features that come** with being part of an Open Source ecosystem_

Problem solved with linguistic engineering 😆

Newdle is part of the [MALT project](https://malt.web.cern.ch/malt/).

![A sneak peek of Newdle](images/sneakpeek.gif)

## Main features

- Add participants searching by name and/or email address
- Choose time slots for different days
- Copy time slots from previous day
- Keep list of participants private
- Check if I am part of any newdles
- Check my newdles

> <br><a href="https://home.cern"><img src="https://raw.githubusercontent.com/indico/assets/master/cern_badge.png" width="64"></a>

Reviewer: - Add the file to this repo instead of hotlinking from another one
- Scale it down so it can be embedded using markdown
- You probably don't need the HTML `<br>` but can use the normal markdown linebreaks

Author: All done except the image resized with markdown. It seems  [it doesn't work for github](https://github.com/hakimel/reveal.js/issues/1349)

Reviewer: what i meant is resizing the actual image file so you don't need to specify a width in markdown

Author: Done

# Newdle

**Newdle** is a tool developed by the [Indico Team](https://getindico.io/about/) @ [CERN](https://home.cern/) to **manage the process of organizing meetings**. As other commercial and Open Source solutions, Newdle provides the functionality of creating ad-hoc polls to schedule meetings when everybody (or at least most of the people involved) are able to attend. How is it possible? The tool provides for the creators of the polls (henceforth **newdles**) the possibility to choose different options to schedule the meeting, i.e. to choose different time slots within a day or from several days, in order to find the best options for everybody.

<img src="images/logo_color.svg" alt="hi" width="300" align="center" />

Newdle has been created to streamline the process of choosing **the perfect date and time** for a meeting or event, avoiding the tedious and time consuming tasks that supposes organizing these meetings when is doing sending emails or private messages.

> Once the newdle is done, the creator **send the invitation to vote** to all the participants s/he wants. Those participants will receive and email with the link of the newdle to vote and they can **choose those option/s that fit** better for them. Once everyone has voted or the creator closes the newdle it is **easy to choose the final date for the event**.

Reviewer 3: ```suggestion
Once the newdle is done, the creator **sends the invitation to vote** to all the participants they want. Those participants will receive an email with the link of the newdle to vote and they can **choose those option(s) that fit** better for them. Once everyone has voted or the creator has closed the newdle it is **easy to choose the final date for the event**.
```

Author: Done :)

> One of the **most remarkable features** of this tool among others is the **user-friendly and modern interface** and also the additional **freedom and flexibility** that come with being part of an Open Source ecosystem. Additionally, none of them seem to seamlessly integrate with other enterprise systems.

Reviewer 2: ```suggestion
One of the **most remarkable features** of this tool among others is the **user-friendly and modern interface** and also the additional **freedom and flexibility** that come with being part of an Open Source ecosystem. Additionally, none of the other options available in the market seem to seamlessly integrate with other enterprise systems.
```

Author: Done

Newdle is part of the [MALT project](https://malt.web.cern.ch/malt/).

![A sneak peek of Newdle](images/sneakpeek.gif)

## Main features

- Add participants searching by name and/or email address
- Choose time slots for different days
- Copy time slots from previous day
- Keep list of participants private
- Check if I am part of any newdles
- Check my newdles

> |||

Reviewer 2: If we want to keep this thing, we should remove the table. I would just remove it for now.

Author: Table removed

One of the **most remarkable features** of this tool among others is the **user-friendly and modern interface** and also the additional **freedom and flexibility** that come with being part of an Open Source ecosystem. Additionally, none of them seem to seamlessly integrate with other enterprise systems.

Newdle is part of the [MALT project](https://malt.web.cern.ch/malt/).

![A sneak peek of Newdle](images/sneakpeek.gif)

## Main features

- Add participants searching by name and/or email address
- Choose time slots for different days
- Copy time slots from previous day
- Keep list of participants private
- Check if I am part of any newdles
- Check my newdles

#### Note

> In applying the MIT license, CERN does not waive the privileges and immunities
> granted to it by virtue of its status as an Intergovernmental Organization

> > or submit itself to any jurisdiction.

Reviewer: I don't think we needs this on the site

Author: Removed
